rover/source/docs/sailboat-hardware.rst

```diff
@@ -19,10 +17,62 @@ to the wind speed the more important this is. See :ref:`wind-vane`.
+
+When using a sail winch the sail limits angle must be configured. 
+This can be tested by arming in manual mode; the throttle will directly control the sail position.
+Throttle stick down (towards you) should result in the sail being sheeted in towards the centre line.
+If the sail is sheeted out the servo should be reversed. The servo min and max parameters can then be
+used to set up the range of travel. The min and max values should be set such that the boom is brought
+in towards the centre line of the boat but not pulled down tightly. The boom should be able to be let
+out until it reaches the shrouds; if no shrouds are fitted the boom shouldn’t go too far past 90 degrees
+to the boat centre line. 
+
+The sail angle :ref:`SAIL_ANGLE_MIN<SAIL_ANGLE_MIN>` and :ref:`SAIL_ANGLE_MAX<SAIL_ANGLE_MAX>` parameters should be set
+to the angle to boom is to the centre line at each extreme of its travel. This allows the angle of the
+boom to be calculated at any point between. 
+
+The ideal sail angle to the wind :ref:`(SAIL_ANGLE_IDEAL)<SAIL_ANGLE_IDEAL>` should then be set. This defines the angle
+between the boom and the wind direction as reported by the wind vane. An angle of zero here would result
+in the boom staying parallel to the wind vane. The boom will keep this angle to the wind until it reaches
+either its minimum or maximum limit. If the sails are too loose this number should be increased. This can
+be tested in ACRO mode. 
+
+Wing sail elevator
+------------------
+A wing sail may be actuated by a secondary control surface akin to a plane's elevator. This should be setup such that the servo mid point (``SERVOx_TRIM``) results in zero angle of attach on the main wing.
+See `Karl's build <https://discuss.ardupilot.org/t/sailboat-support/32060/321>`__ . Use servo output function, ``SERVOx_FUNCTION``, 128 - Wing Sail Elevator.
+
+The servo min (``SERVOx_MIN``) and max (``SERVOx_MAX``) end points should be set such that the main wing is held in a maximum lift angle of attack on both tacks.
+If the wing sail is acting on the wrong tack the servo should be reversed with ``SERVOx_REVERSE``. The `SAIL_ANGLE_*` parameters are not use with wing sail elevators.
+
+Direct wing sail actuation
+--------------------------
+Wing sail is directly driven by a servo, in this case use servo output function, ``SERVOx_FUNCTION``, 137 - Sail Mast Rotation. See `Ari's build <https://github.com/ArduPilot/ardupilot/pull/17066#issuecomment-813692335>`__ 
+
+The servo mid point (``SERVOx_TRIM``) must be set such such that the wing is held on the center line of the boat. The min (``SERVOx_MIN``) and max points (``SERVOx_MAX``) should be set
+such that the wing moves to the same angle on each tack. This maximum angle should be measured and set to the :ref:`SAIL_ANGLE_MAX<SAIL_ANGLE_MAX>` parameter. As with sheeted sails :ref:`SAIL_ANGLE_IDEAL<SAIL_ANGLE_IDEAL>`
```

Comment:
I have added a comment that close to 90 deg is better. I think we should actually work fine for angles above 90 just that it will then sheet back in to 90 deg when switching to 'drag' sailing.